pkgs/applications/editors/emacs-modes/recipes-archive-melpa.json

```diff
@@ -55255,11 +55846,11 @@
   "repo": "immerrr/lua-mode",
   "unstable": {
    "version": [
-    20190113,
-    1050
+    20191015,
+    733
    ],
-   "commit": "95c64bb5634035630e8c59d10d4a1d1003265743",
-   "sha256": "0cawb544qylifkvqads307n0nfqg7lvyphqbpbzr2xvr5iyi4901"
```

Comment:
`nixos-rebuild switch --upgrade` still fails trying to find something with this sha. Should it really take five days for the merge to trigger a channel update?

Reply:
The channels advance when Hydra builds a substantial subset of packages. (Because people hate building stuff on their computers. Especially heavy beasts like browsers, libreoffice, etc.) That's why it takes so long.

This fix has already made to `nixos-19.09-small` channel—this is the channel that waits for fewer packages to build, so it usually moves faster.

Reply:
Thanks for the pointer, I might move to -small for a while.